sagemaker_batch_transform/tensorflow_open-images_tfrecord/tensorflow-serving-tfrecord.cli.ipynb

```diff
@@ -39,15 +39,6 @@
     "region = sagemaker_session.boto_region_name\n",
```

Comment:
<p><code>uri_suffix</code>  and <code>account_id</code>  were not defined in the cells above. So when you parse it the bash script, it raises an exception. Easy fix: simply remove these variables and remove <code>URI_SUFFIX=$5</code>  and <code>ACCOUNT_ID=$6</code>  since you never used these variables in the script. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1906/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
fixed